### ISSUE TITLE
Uncompressed version of blockly

### ISSUE THREAD

**Issue by pantoninho** [mo4islona/node-blockly#20]
I've been wrapping Blockly inside a npm package, and it has not been easy.

I successfully wrapped the compressed version of `Blockly`, but I would like to be able to use the uncompressed version for development/debugging. I've hit a wall with the `google-closure-library` dependency and started searching the web.

Looking at your project, I've noticed you are also using the compressed files... have you tried using the uncompressed version? If yes, do you think is possible?

Thank you

**Comment by eryckpedro** [comment]
@pantoninho are you having troubles using node js and blockly? I've been trying to use both of them and hit the same wall with the `google-closure-library`. I'm using the uncompressed files and, to get it working, you need to install both of them via the `npm` command (or you can copy&paste the folders to the node_modules folder, but be sure to let them named "blockly" and "google-closure-library")

PS: I recommend cloning blockly and adding it manually, because the project workers don't actually manage the npm dependencies to check if it's up to date.

**Comment by pantoninho** [comment]
@eryckpedro Yes, it has been very hard to get Blockly working with commonjs/npm... It has already crossed my mind some code-mod to replace every goog reference...
I'm going to try your solution tomorrow, looks promising!

Thank you

**Comment by pantoninho** [comment]
@eryckpedro I think it's pretty much a dead end.. I'm going to try something else: use an offline version of google-closure-compiler..

**Comment by eryckpedro** [comment]
@pantoninho TBH that's what I'm using in my project. It seems that when you finish using and creating your files using the uncompressed library, you run a `build.py` in `Blockly's` root directory that [compresses](https://developers.google.com/blockly/guides/modify/web/building) the modified files you created.

**Comment by prabak** [comment]
Hi,

I am also trying to work with the uncompressed version of google blockly. I am not sure if I understood how to use it with node-blockly. 

yarn add node-blockly
copy blockly and closure library to node_modules/

Then where would I modify to allow for it to use the uncompressed version of blocks and blockly from google blockly? From what I understand, node-blockly uses gulp to modify the files.

thanks
